templates/zinst/0/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '2'
+services:
+  zinst-repository:
+    image: zinst/zinst_repository:0.4
+    ports:
+    - 8080:80/tcp
```

Comment:
Port it's not published with ${public_port} value.
Anyway, better to use lb to publish ports, don't needed. 
```
-    ports:
-    - 8080:80/tcp
```